doc/GuideForCommitters.md

```diff
@@ -93,19 +93,19 @@ build, add a comment to the pull request that follows the syntax:
 Where `{platform-list}` is a comma-separated list of platforms chosen from the
 following tags:
 
-| Tag      | Platform |
-| :----    | :------- |
-| zlinux   | 64-bit Linux on Z |
-| zos      | 64-bit zOS |
-| plinux   | 64-bit Power Linux LE |
-| aix      | 64-bit AIX |
-| x32linux | 32-bit Linux |
-| xlinux   | 64-bit Linux |
-| xcrlinux | 64-bit Compressed References Linux |
-| win      | 64-bit Windows |
-| osx      | 64-bit macOS |
-| arm      | 32-bit AArch32 (cross-compile build only) |
-| aarch64  | 64-bit AArch64 (cross-compile build only) |
+| Tag      | Platform                                           | Alias               |
```

Comment:
`Tag` and `Alias` is a bit confusing to me because we also use `alias` in `omrbuild.groovy` which has a slightly different meaning. I would maybe change Tag to Alias and Alias to Spec Name?

Reply:
Updated.